tools/runners/error-page/event-handler.js

```diff
@@ -0,0 +1 @@
+../../../packages/build-events/event-handler.js
```

Comment:
I couldn't think of another way to share code between the tool and the `build-events` package… :thinking: